### DISCUSSION TITLE
Could artifact retrieval include messages from the hosts?

### DISCUSSION THREAD

**Discussion by bjhartin** [sbt/sbt#6451] [Ideas]
Given yesterday's bintray brownout, I wonder if it would be helpful for artifact resolution to include an additional GET to a /messages endpoint on the hosts.  If a 200 is returned, this message could be displayed in the SBT output, e.g.

```
[warn] WARNING: There will be a brownout on April 12th and 26th to remind you that Bintray is being retired on May 1st.  This can break CI pipelines that:

a) spin up new agents which won't have a local cache (e.g. Drone)
b) don't use a proxy repository to cache retrieved artifacts
c) rely on artifacts that are only available in Bintray (e.g. some common sbt plugins)
```

**Comment by eed3si9n** [comment]
I don't know if that's a great idea. For the most part the users can't do much about the situation, and it's up to the repository admins to mitigate these situations, and we're normally aware of it.
